src/pykeen/models/multimodal/base.py

```diff
@@ -2,30 +2,43 @@
 
 """Base classes for multi-modal models."""
 
-from class_resolver.utils import OneOrManyHintOrType, OneOrManyOptionalKwargs
+from typing import ClassVar, Tuple, Type
+
+import torch
+from class_resolver import HintOrType, OneOrManyHintOrType, OneOrManyOptionalKwargs, OptionalKwargs
 
 from ..nbase import ERModel
+from ...nn.combinations import Combination
 from ...nn.init import PretrainedInitializer
-from ...nn.modules import LiteralInteraction
-from ...nn.representation import Embedding, Representation
+from ...nn.modules import Interaction
+from ...nn.representation import CombinedRepresentation, Embedding, Representation
 from ...triples import TriplesNumericLiteralsFactory
-from ...typing import HeadRepresentation, RelationRepresentation, TailRepresentation
 from ...utils import upgrade_to_sequence
 
 __all__ = [
     "LiteralModel",
 ]
 
 
-class LiteralModel(ERModel[HeadRepresentation, RelationRepresentation, TailRepresentation], autoreset=False):
+class LiteralModel(
+    ERModel[
+        Tuple[torch.FloatTensor, torch.FloatTensor], torch.FloatTensor, Tuple[torch.FloatTensor, torch.FloatTensor]
+    ],
+    autoreset=False,
+):
     """Base class for models with entity literals that uses combinations from :class:`pykeen.nn.combinations`."""
 
+    #: the interaction class (for generating the overview table)
+    interaction_cls: ClassVar[Type[Interaction]]
+
     def __init__(
         self,
         triples_factory: TriplesNumericLiteralsFactory,
-        interaction: LiteralInteraction,
+        interaction: HintOrType[Interaction[torch.FloatTensor, torch.FloatTensor, torch.FloatTensor]],
```

Comment:
you want `InstOrType` here since this can't be none

Reply:
cf. https://github.com/pykeen/pykeen/pull/964#discussion_r892002036